arch/esp32/esp32.ini

```diff
@@ -1,7 +1,7 @@
 ; Common settings for ESP targes, mixin with extends = esp32_base
 [esp32_base]
 extends = arduino_base
-platform = platformio/espressif32@^6.4.0
+platform = platformio/espressif32@6.3.2
```

Comment:
Change is fine, i'd recommend adding a comment to look for the root cause

Reply:
Done